circle.yml

```diff
@@ -248,7 +248,7 @@ commands:
           working_directory: ~/build
           command: |
             if [ ! -d wasm-spec ]; then
-              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-tests-backported-20201113 --depth 1 wasm-spec
+              git clone https://github.com/wasmx/wasm-spec --branch w3c-1.0-tests-backported-fizzy-0.7.0 --depth 1 wasm-spec
```

Comment:
I think we agreed to use datestamp for the branches/tags, i.e. 3c-1.0-tests-backported-20210112 or whatever is the date of the last included commit.

Reply:
Yeah this is a temporary branch that will be merged into `w3c-1.0-tests-backported` branch and then tagged `w3c-1.0-tests-backported-<date>` when we make a release (or could be earlier).

Reply:
Ah okay. I think we should aim to make a release next week.